Makefile

```diff
@@ -64,7 +64,7 @@ install-host: proto-host
 
 # used to build under Docker
 proto-host: $(PROTOFILES)
-	@for DIR in $(DIRS); do cd $${DIR}; ls *.proto > /dev/null 2>&1 && protoc *.proto --go_out=plugins=grpc:.; done
+	@for DIR in $(DIRS); do cd $(BASEDIR)/$${DIR}; ls *.proto > /dev/null 2>&1 && protoc *.proto --go_out=plugins=grpc:. || true; done
```

Comment:
You must not have rebased with master before submitting this PR? My previous PR incorporated this and it's in the `master` branch.